dotnet/src/Plugins/Plugins.UnitTests/Web/Bing/BingTextSearchTests.cs

```diff
@@ -25,6 +27,20 @@ public BingTextSearchTests()
         this._kernel = new Kernel();
     }
 
+    [Fact]
+    public void AddBingTextSearchSucceeds()
+    {
+        // Arrange
+        var builder = Kernel.CreateBuilder();
+
+        // Act
+        builder.AddBingTextSearch(apiKey: "ApiKey");
+        var kernel = builder.Build();
+
+        // Assert
+        Assert.NotNull(kernel.Services.GetRequiredService<ITextSearch>());
```

Comment:
nit: The `GetRequiredService` method will never return null. As I understand it, it either returns the requested service or throws an exception. Perhaps you can check if the service returned by the method is of type `BingTextSearch`.